zipline/finance/performance/tracker.py

```diff
@@ -401,17 +397,16 @@ def handle_minute_close(self, dt):
         todays_date = normalize_date(dt)
         account = self.get_account(True)
 
-        minute_returns = self.minute_performance.returns
         self.minute_performance.rollover()
-        # the intraday risk is calculated on top of minute performance
-        # returns for the bench and the algo
-        self.intraday_risk_metrics.update(dt,
-                                          minute_returns,
-                                          self.all_benchmark_returns[dt],
-                                          account)
 
-        bench_since_open = \
-            self.intraday_risk_metrics.benchmark_cumulative_returns[dt]
+        # Duplicate intraday_risk_metrics work of calculating the benchmark
+        # returns since open.
+        # intraday_risk_metrics is marked for removal.
+        #
+        # This redundant work is in anticipation of no longer being able to
+        # depend on the 'since open' calculations in intraday_risk_metrics.
+        bench_returns = self.all_benchmark_returns.loc[todays_date:dt]
+        bench_since_open = (1. + bench_returns).prod() - 1
```

Comment:
I should (and will) update the comment before merging, since the actual removal is done in this branch.

I think the factoring out is possibly half of one, six of the other kind of call. Since it is a pretty standard one liner for getting the cumulative returns... but can see the benefits of clearly labeling it.

For now, I think I'm going to leave it, but with a `# cumulative returns` comment.